pkg/httputil/http.go

```diff
@@ -104,3 +113,48 @@ func newError(resp *http.Response) *Error {
 func (e *Error) Error() string {
 	return fmt.Sprintf("API error (%d): %s", e.Status, e.Message)
 }
+
+func DownloadResponseToFile(sourceURL, path string) {
+	file, err := fileutil.CreateFile(path)
+	printutil.LogFatal(err)
+
+	client := http.Client{
+		CheckRedirect: func(r *http.Request, via []*http.Request) error {
+			r.URL.Opaque = r.URL.Path
+			return nil
+		},
+	}
+	resp, err := client.Get(sourceURL) //nolint
+	defer func(Body io.ReadCloser) {
+		_ = Body.Close()
+	}(resp.Body)
+	printutil.LogFatal(err)
+
+	err = fileutil.WriteToFile(path, resp.Body)
+	printutil.LogFatal(err)
+
+	defer func(file *os.File) {
+		_ = file.Close()
+	}(file)
+	logrus.Infof("Downloaded %s from %s", path, sourceURL)
+}
+
+func RequestAndGetJSONBody(route string) map[string]interface{} {
+	res, err := http.Get(route) //nolint
+	defer func(Body io.ReadCloser) {
+		_ = Body.Close()
+	}(res.Body)
```

Comment:
```suggestion
	defer res.Body.Close()
```
nit: simplifying the code